state/processList.go

```diff
@@ -1169,6 +1169,11 @@ func NewProcessList(state interfaces.IState, previous *ProcessList, dbheight uin
 		pl.DirectoryBlock = directoryBlock.NewDirectoryBlock(nil)
 		pl.AdminBlock = adminBlock.NewAdminBlock(nil)
 		pl.EntryCreditBlock, err = entryCreditBlock.NextECBlock(nil)
+
```

Comment:
Wondering if there are any other side effects like this for the Admin & Entry Credit Blocks

Reply:
@PaulSnow I think there may be a few options - should we store more information about these Blocks in the Fastboot save file or try to retrieve db entries to repopulate